template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Controllers/HomeController.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity.Service;
+using Microsoft.AspNetCore.Mvc;
+using Company.WebApplication1.Identity.Models.HomeViewModels;
+using Microsoft.Extensions.Configuration;
+
+namespace Company.WebApplication1.Identity.Controllers
+{
+    [Area(IdentityServiceConstants.Tenant)]
+    [Route(IdentityServiceConstants.RoutePrefix + "/[area]")]
+    [Authorize(IdentityServiceOptions.LoginPolicyName), AllowAnonymous]
+    public class HomeController : Controller
+    {
+        public HomeController(
+            ApplicationManager<IdentityServiceApplication> appManager,
+            IConfiguration config)
+        {
+            ApplicationManager = appManager;
+            Configuration = config;
+        }
+
+        public ApplicationManager<IdentityServiceApplication> ApplicationManager { get; set; }
+        public IConfiguration Configuration { get; set; }
+
+        [HttpGet]
+        public IActionResult Index()
+        {
+            var model = new IdentityServiceViewModel()
+            {
+                Issuer = Configuration["Authentication:Identity:Issuer"],
```

Comment:
Should you put this into an ApplicationOptions and bind config against it instead of directly using config?